applications/FluidDynamicsApplication/tests/cpp_tests/test_embedded_skin_visualization_process.cpp

```diff
@@ -121,85 +168,143 @@ namespace Kratos {
             skin_visualization_process.ExecuteBeforeOutputStep();
             skin_visualization_process.ExecuteFinalizeSolutionStep();
 
-            GidIO<> gid_io_fluid("/home/rzorrilla/Kratos/tests/visualizaton_model_part_2d", GiD_PostAscii, SingleFile, WriteDeformed, WriteConditions);
-            gid_io_fluid.InitializeMesh(0);
-            gid_io_fluid.WriteMesh(visualization_model_part.GetMesh());
-            gid_io_fluid.FinalizeMesh();
-            gid_io_fluid.InitializeResults(0, visualization_model_part.GetMesh());
-            gid_io_fluid.WriteNodalResults(VELOCITY, visualization_model_part.Nodes(), 0, 0);
-            gid_io_fluid.WriteNodalResults(PRESSURE, visualization_model_part.Nodes(), 0, 0);
-            gid_io_fluid.FinalizeResults();
-        }
+            // Check values
+            KRATOS_CHECK_NEAR(visualization_model_part.GetNode(1).FastGetSolutionStepValue(PRESSURE), -1.0, 1e-8);
```

Comment:
what happens if you ask to interpolate VELOCITY_X? ( a component)

Reply:
It can be done. The process works with scalar, vector and component variables.